### DISCUSSION TITLE
Posibilidad de incluir icono Bizum a Mistica Icons

### DISCUSSION THREAD

**Discussion by RPalis** [Telefonica/mistica-design#1065] [Proposals]
Hola, equipo. ¿Podríais incluir el icono de Bizum en la próxima actualización de la Biblioteca Mística? 
Este icono toca por ahora en Moves, O2ES, Telefónica y Solar. 

[- Figma](https://www.figma.com/file/le5U4x1qHCF0Bg8TfF5IEl/Bizum---Moves---UX-UI?node-id=5398%3A7232&t=wzAoKy6eSoUebkDV-4)

**Comment by lauraocana** [comment]
¿Sería únicamente para DiPa en este momento @RPalis? Para que tengan toda la info. Thanks!

**Reply by RPalis** [reply]
Voy Investigarlo mejor pero ahora es una petición de DIPA.

**Comment by yceballost** [comment]
Actualmente, en la librería de iconos de Mística solo admitimos iconos.

En algún momento hemos pensado que podría estar bien tener una pequeña librería de "asset"para este tipo de peticiones. Por ejemplo: logo Bizum, mastercard, visa, logos de netflix y otros servicios, etc.

No sé hasta que punto tendría sentido.

De momento, para añadir el logo de bizum a algún diseño tendría que exportarse manualmente y pasarlo al desarrollador

**Reply by RPalis** [reply]
Vale muchas gracias por el contexto y cual serian los standards para que lo pudiera incluir estos assets a Mística?  
Pues hay una librería de VIVO DIPA por ej.  Es possible quizás crear una especie de extension Mística DIPA?

**Reply by yceballost** [reply]
pues es una decisión del proyecto Mística si merece la pena albergar ese tipo de assets en el sistema, de momento es la primera vez que llega esta necesidad.

En este caso crear una librería extended de DIPA no tiene mucho sentido solo para esto y de momento yo creo que podría vivir en la librería de TCX, suficiente que la librería TCX no tiene dueño... mejor no crear una nueva 😂

**Reply by RPalis** [reply]
Vale Yayo, muchas gracias por vuestra orientación. [🧑‍🚀](https://emojipedia.org/astronaut/)

**Comment by yceballost** [comment]
hablando con el equipo, de momento no lo consideramos como una necesidad global porque es la primera vez que nos llega esto por parte de los equipos, como era una idea que rondaba en nuestras cabezas y además es algo que puede ser util en el futuro, no descartamos que más adelante podamos incluir este tipo de assets en mística, de momento lo gestiona cada equipo.
